website/content/docs/enterprise/index.mdx

```diff
@@ -12,35 +12,51 @@ features address the organizational complexities of collaboration, operations, s
 If you have purchased or wish to try out Consul Enterprise,
 refer to [how to access Consul Enterprise](#access-consul-enterprise).
 
-## Enterprise Features
+## Enterprise features
 
 The following features are [available in several forms of Consul Enterprise](#consul-enterprise-feature-availability).
 
 ### Multi-Tenancy
+
 - [Admin Partitions](/consul/docs/enterprise/admin-partitions): Define administrative boundaries between tenants within a single Consul datacenter
 - [Namespaces](/consul/docs/enterprise/namespaces): Define resource boundaries within a single admin partition for further organizational flexibility
 - [Sameness Groups](/consul/docs/connect/config-entries/samenes-group): Define partitions and cluster peers as members of a group with identical services
 
 ### Resiliency
+
 - [Automated Backups](/consul/docs/enterprise/backups): Configure the automatic backup of Consul state
 - [Redundancy Zones](/consul/docs/enterprise/redundancy): Deploy backup voting Consul servers to efficiently improve Consul fault tolerance
 
 ### Scalability
+
 - [Read Replicas](/consul/docs/enterprise/read-scale): Deploy non-voting Consul servers to enhance the scalability of read requests
 
-### Operational Simplification
+### Operational simplification
+
 - [Automated Upgrades](/consul/docs/enterprise/upgrades): Ease upgrades by automating the transition from existing to newly deployed Consul servers
 - [Consul-Terraform-Sync Enterprise](/consul/docs/nia/enterprise): Leverage the enhanced network infrastructure automation capabilities of the enterprise version of Consul-Terraform-Sync
 
-### Complex Network Topology Support
+### Complex network topology support
+
 - [Network Areas](/consul/docs/enterprise/federation): Support complex network topologies between federated Consul datacenters with pairwise federation rather than full mesh federation
 - [Network Segments](/consul/docs/enterprise/network-segments/network-segments-overview): Support complex network topologies within a Consul datacenter by enforcing boundaries in Consul client gossip traffic
 
 ### Governance
+
 - [OIDC Auth Method](/consul/docs/security/acl/auth-methods/oidc): Manage user access to Consul through an OIDC identity provider instead of Consul ACL tokens directly
 - [Audit Logging](/consul/docs/enterprise/audit-logging): Understand Consul access and usage patterns by reviewing access to the Consul HTTP API 
 - [Sentinel for KV](/consul/docs/enterprise/sentinel): Policy-as-code framework for defining advanced key-value storage access control policies
 
+## Regulatory compliance
+
+- [FIPS 140-2 Compliance](/consul/docs/enterprise/fips): Leverage FIPS builds of Consul Enterprise to ensure your Consul deployments are secured with BoringCrypto and CNGCrypto, and compliant with FIPS 140-2.
+
+  <Note>
+
+  FIPS 140-2 builds of Consul Enterprise support all runtimes (VMs, Kubernetes, ECS) except for Lambda. In addition, HCP does not currently support FIPS builds of Consul Enterprise.
```

Comment:
I'll leave it up for @Jeff-Apple to confirm but I believe ECS does not support FIPS yet.

Reply:
I'm waiting to hear back from the engineer as to why we haven't published the FIPS version of consul-ecs yet. It is supposed to be supported.

Reply:
ECS isn't supported for now. It will be the next time we release a new version of consul-ecs.